torch_xla/csrc/xla_backend_impl.cpp

```diff
@@ -151,6 +158,10 @@ class XlaBackendImpl : public torch::lazy::BackendImplInterface {
     return at::DeviceType::CPU;
   }
 
+  void SetDefaultDeviceType(std::string type) override {
+    default_device_type_ = XlaDeviceType(c10::Device(type).type());
```

Comment:
nit: is the `c10::Device(type).type()` always same as our own definition of `XlaDeviceType` as defined in https://github.com/pytorch/xla/blob/master/torch_xla/csrc/device.h#L13?

Reply:
Ah OK, I guess the right thing to do here is to get a defualt device and set the default type correctly. In the TPU setup we actually want it to be TPU. I will update this.

Reply:
hmm, wait. This function already takes string as type. What's the right thing to do here? How do I convert `string` to `XlaDeviceType`?

Maybe we need to convert  https://github.com/pytorch/xla/blob/dbc67604cfff305a37544513c20e377b254054ef/torch_xla/csrc/device.cpp#L53-L64

to a helper function? @wonjoolee95

Reply:
What does the `c10::Device(type).type()` return? Is it just the type of the device in string format like `TPU` or `CPU`? If so, I think you're right we can move this to a helper function.